app/src/main/java/nu/yona/app/ui/YonaActivity.java

```diff
@@ -349,11 +349,12 @@ public void onResume()
 
 	private void checkForNotificationPermission()
 	{
-		if (!NotificationManagerCompat.from(this).areNotificationsEnabled())
+		if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O || (NotificationManagerCompat.from(this).areNotificationsEnabled() && AppUtils.arePersistentNotificationsEnabled(getActivity(), AppConstant.YONA_SERVICE_CHANNEL_ID)))
```

Comment:
I should have asked in the first review round: are you sure the persistent notification needs to be enabled for Yona to work properly?

Reply:
Yes, from oreo and above without a persistent notification activity monitoring service doesn't work.

Reply:
In that case, please make a PR for ``master`` too with the same change set. For the error message, I felt it wasn't worth it, but this part makes it worthwhile.